syntax = "proto3";

message EventDetail {
  string name = 1;
  string event_type = 2;
  bytes details = 3;
}